### PR TITLE
Added a RichText ediror

### PR DESCRIPTION
**Pull request by Fredrick-Waihenya**
# Description
I added ckeditor for the blog model content field.

### CODE REVIEW

**Review by JimmyTron — CHANGES_REQUESTED** [review]
Too many files changed. Its a possible gateway to introduce bugs. 
I would recommend you make small changes and modularize the changes. 
Also open an issue of what you want to tackle or request someone to create it. 
Also ensure you check on these tests so that they dont fail. 
If they fail it means the change breaks something.

**Review by jumalaw98 — CHANGES_REQUESTED** [review]
Check on why there's an error
Screenshot from 2023-01-12 17-08-30
, on the ci test

File: requirements.txt [hunk 1]
```diff
@@ -1,15 +1,562 @@
...
+yarl==1.7.2
+yaswfp==0.9.3
+zc.lockfile==2.0
+zipp==1.0.0
+zlib-wrapper==0.1.3
+zope.component==5.0.1
+zope.deprecation==4.4.0
+zope.event==4.4
+zope.hookable==5.1.0
+zope.interface==5.4.0
```
Comment (JimmyTron) [inline requirements.txt]: You haven't used a vitual enviroment to run this project hence the enormous number of mudules coming from your pip freeze. 
The changes you made don' t warrant installing all these packages to our server machine. 
to mitigate this clone the project onto a new location the create a virtual environment. from there you activate the env and install dependancies.
This is the source of the error @jumalaw98 noted earlier.  also check on pr no #74 for further clarification

**Review by JimmyTron — CHANGES_REQUESTED** [review]
Check on the highlighted area

**Comment by Fredrick-Waihenya** [comment]
I just realized I had not activated my virtual environment before running pip.

**Review by JimmyTron — APPROVED** [review]
Nice work
We'll still need you to do a code review session for the change

**Review by jumalaw98 — APPROVED** [review]

**Pull request merged**